test/e2e/data/infrastructure-azure/v1beta1/cluster-template-kcp-adoption/step1/cluster-with-cp0.yaml

```diff
@@ -50,7 +50,7 @@ spec:
       storageAccountType: Premium_LRS
     osType: Linux
   sshPublicKey: ${AZURE_SSH_PUBLIC_KEY_B64:=""}
-  vmSize: Standard_D2s_v3
+  vmSize: Standard_B2s
```

Comment:
can we avoid changing the kcp adoption test and related files? It's all getting removed shortly in #2731